dom.bs

```diff
@@ -1139,9 +1216,15 @@ invoked, must run these steps:
    <var>listener</var>'s <b>capture</b> is true, terminate these substeps (and run them for the next
    <a>event listener</a>).
 
+   <li>If <var>listener</var>'s <b>passive</b> is true, set <var>event</var>'s <a>in passive listener flag</a>.
+
    <li><p>Call <var>listener</var>'s <b>callback</b>'s {{EventListener/handleEvent()}}, with
    <var>event</var> as argument and <var>event</var>'s {{Event/currentTarget}} attribute value as
-   <a>callback this value</a>. If this throws any exception, <a>report the exception</a>.
+   <a>callback this value</a>.
+
+   <li>Clear <var>event</var>'s <a>in passive listener flag</a>.
+
+   <li>If the call to {{EventListener/handleEvent()}} threw any exception, <a>report the exception</a>.
```

Comment:
I just wasn't thinking about it properly (thinking in terms of an algorithm with a 'throw' statement).  Thanks for catching this, I agree it's better and I don't anticipate any implementation issue.